_scripts/instruction-widget/templates/getting-started/webroot.html

```diff
@@ -0,0 +1,20 @@
+<p>
+Certbot supports a number of different “plugins” that can be used to obtain and/or install certificates.
+
+Since your server architecture doesn't yet support automatic installation you should use the <a href="https://certbot.eff.org/docs/using.html#webroot">webroot command</a> to obtain your certificate.
```

Comment:
Just because we can't automatically install their cert doesn't necessairly mean they should use `webroot`. They may instead want to use `standalone`.

While most of this file looks good, I think we should have wording here closer to the [original certonly.html file](https://github.com/certbot/website/blob/d9c6d84bdbd67f62d775db6084a9c1f22933e194/_scripts/instruction-widget/templates/getting-started/certonly.html) before I messed it up by suggesting they use the UI.